nautobot_device_lifecycle_mgmt/tests/conftest.py

```diff
@@ -148,8 +148,8 @@ def create_validated_softwares():
         software=software_one,
         start=date(2019, 1, 10),
     )
-    validatedsoftwarelcm.device_types.set([device_type])  # pylint: disable=no-member
     validatedsoftwarelcm.save()
```

Comment:
I'm honestly shocked that this (wrong) order of operations worked in Postgres...

Reply:
Me too. Pretty odd behavior all around